californium-core/src/main/java/org/eclipse/californium/core/observe/InMemoryObservationStore.java

```diff
@@ -36,8 +44,33 @@
 
 	private static final Logger LOGGER = LoggerFactory.getLogger(InMemoryObservationStore.class.getName());
 	private final ConcurrentMap<Token, Observation> map = new ConcurrentHashMap<>();
+	private volatile boolean enableStatus;
 
-	public InMemoryObservationStore() {
+	public InMemoryObservationStore(NetworkConfig config) {
+		int healthStatusInterval = config.getInt(NetworkConfig.Keys.HEALTH_STATUS_INTERVAL, 60); // seconds
+		
+		if (healthStatusInterval > 0 && LOGGER.isInfoEnabled()) {
```

Comment:
this is what I meant above regarding the *compatible use of the check for > 0*. However, it seems that we either want to dump the info everywhere or nowhere at all, right?

Reply:
Yes, that's my current intention. This logging is just a util for the "main work", the debugging of the message exchange leak.